compilator/compilator.go

```diff
@@ -471,16 +470,21 @@ func (c *Compilator) compilePackage(pkg *model.Package) (err error) {
 			return color.GreenString("compilation-%s > %s", color.MagentaString("%s", pkg.Name), color.RedString("%s", line))
 		},
 	)
-	exitCode, container, err := c.DockerManager.RunInContainer(
-		containerName,
-		c.BaseImageName(),
-		[]string{"bash", containerScriptPath, pkg.Name, pkg.Version},
-		c.getTargetPackageSourcesDir(pkg),
-		c.getPackageCompiledTempDir(pkg),
-		c.keepContainer,
-		stdoutWriter,
-		stderrWriter,
-	)
+	mounts := map[string]string{
+		c.getTargetPackageSourcesDir(pkg): docker.ContainerInPath,
+		c.getPackageCompiledTempDir(pkg):  docker.ContainerOutPath,
+		"source_mount":                    "/var/vcap/source/",
```

Comment:
It's a temporary docker volume, which lives wherever docker wants it to live.  Usually, it's in `/var/lib/docker/volumes/` somewhere. `source_mount` is just a name to figure out volumes.  (I actually munge it in the callee to include the container name… also happy to stick in a UUID or something to avoid reuse.)

Yeah, we should add a constant for that.  Was just hurrying up the thing to make sure it actually fixes the problems, first.